File: sample_src/services/orders_cs/InvoiceController.cs
namespace Sample.Invoices;

/// <summary>HTTP entry point for invoices.</summary>
[ApiController]
[Route("cs/[controller]")]
public class InvoiceController : ControllerBase
{
    private readonly InvoiceService _service;

    public InvoiceController(InvoiceService service)
    {
        _service = service;
    }

    /// <summary>Issue an invoice for one order.</summary>
    [HttpPost]
    public Invoice Create(InvoiceRequest request)
    {
        return _service.Issue(request);
    }

    /// <summary>Fetch one invoice by id.</summary>
    [HttpGet("{id}")]
    public Invoice Find(string id)
    {
        return _service.Find(id);
    }
}

File: tests/fixtures/langs/laravel/app/Http/Controllers/InvoiceController.php
<?php

/** Invoices, routed by `Route::resource('invoices', ...)->only(['index', 'show'])`. */
class InvoiceController
{
    /** List invoices. */
    public function index()
    {
        return [];
    }

    /** Show one invoice. */
    public function show($invoice)
    {
        return $invoice;
    }
}
